doc/Extensions/Applications.md

```diff
@@ -631,6 +631,82 @@ The application should be auto-discovered as described at the top of
 the page. If it is not, please follow the steps set out under `SNMP
 Extend` heading top of page.
 
+## Add Custom Linux Sensors
+Adding unsupported sensors and states as extend SNMP attribute.
+
+You can extend SNMP to LNMS custom sensors using your own scripts.
+This guide will walk you through the process of creating, configuring,
+and using a custom SNMP extension.
+
+1. Create Your Custom Script
+
+Write a script in your preferred programming language (e.g., Python, Bash)
+to fetch sensor data. Save the script in a directory like /etc/snmp/.
+
+The script should output sensor data in the following format:
+
+[unique_name1],[sensor_type],[description1],[low_limit],[low_warn_limit],[high_warn_limit],[high_limit],[group];
```

Comment:
Since formatting is important, it would these bit right 4 lines should be made enclosed between ```.

Reply:
Thanks for your review! After we a decision how we implement this interface I will rewrite the doc.